src/source/status_events.cr

```diff
@@ -39,15 +42,30 @@ module PlaceOS::Source
     end
 
     def stop
+      update_values
+
       self.stopped = true
+
       return unless @redis
       begin
         redis.punsubscribe(STATUS_CHANNEL_PATTERN)
       rescue
       end
+
       redis.close
     end
 
+    def update_values
+      PlaceOS::Model::Module.all.in_groups_of(64, reuse: true) do |modules|
+        modules.each do |mod|
+          store = PlaceOS::Driver::RedisStorage.new(mod.try(&.id) || raise Exception.new("Nil assertion failed since the module is Nil"))
```

Comment:
We should probably just skip over anything where mod.try(&.id) == nil
something like:

```crystal
mod_id = mod.try(&.id)
next unless mod_id
store = PlaceOS::Driver::RedisStorage.new(mod_id)
```

Reply:
Agreed, instead of raising that would be a much better outcome